table-api/src/main/java/io/deephaven/api/TableOperations.java

```diff
@@ -1245,4 +1245,24 @@ TOPS updateBy(UpdateByControl control, Collection<? extends UpdateByOperation> o
     TOPS wavgBy(String weightColumn, Collection<String> groupByColumns);
 
     // -------------------------------------------------------------------------------------------
+
+    TOPS ungroup();
+
+    TOPS ungroup(boolean nullFill);
+
+    TOPS ungroup(String... columnsToUngroup);
+
+    TOPS ungroup(boolean nullFill, String... columnsToUngroup);
+
+    /**
+     * Ungroups a table by converting arrays into columns.
+     *
+     * @param nullFill indicates if the ungrouped table should allow disparate sized arrays filling shorter columns with
+     *        null values. If set to false, then all arrays should be the same length.
+     * @param columnsToUngroup the columns to ungroup
+     * @return the ungrouped table
+     */
```

Comment:
This is an excuse to update the JavaDoc and copy it to the other overloads. For the basic case:
```
Ungroups a table by expanding columns of arrays or vectors into columns of singular values, creating one row in the output table for each value in the columns to be ungrouped. Columns that are not ungrouped have their values duplicated in each output row corresponding to a given input row.
```
We want to be clear that it's arrays *and* vectors.

Reply:
Done